.github/workflows/java-common-build.yml

```diff
@@ -0,0 +1,37 @@
+name: Workflow Test/Build
+
```

Comment:
common inputs from java?

 mavenS3AccessKeyId:
    description: 'Maven S3 Access Key Id to upload artefact'
  mavenS3SecretAccessKey:
    description: 'Maven S3 Secret Access Key to upload artefact'

Reply:
This one is going to be converted into an action (rather than a workflow)

Reply:
@cartoncloudthongnguyen please turn this into an action. Basically it is this same file with the following changes:

- Rename to action.yml and move to its own folder called build-library-java 
- Add README.md
- Add description property
- Move inputs property to be a top level property. Make secrets as inputs.
- Replace `jobs` with
`runs:
  using: composite`
- Add `shell: bash` on each mvn steps

See https://github.com/cartoncloud/actions/tree/v3/deploy-java for the service version of this action,